.gitignore

```diff
@@ -19,3 +19,9 @@ js/javascript.min.js
 # these are easily recreated on a new project
 # with `npm install`
 node_modules/
+
+# ignore composer dependencies
+# these are easily recreated on a new project
+# with `composer install`
+vendor/
+composer.lock
```

Comment:
@dosten My understanding is, assuming you are loading dependencies with Composer, including `composer.lock` in the repo is a best practice because rather than installing the latest version of the dependency, you install a version of the dependency that is __known__ to work.  Based on this, I think it makes sense to include `composer.lock`